content/en/docs/features/keepalive.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'KeepAlive'
+description: 'How keepalive works with plgd hub'
+date: '2023-01-10'
+lastmod: '2023-01-10'
+categories: [features]
+keywords: ['keepalive', 'timeout', 'iotivity-lite', 'coap-gateway']
+menu:
+  docs:
+    parent: features
+    weight: 60
+toc: true
+---
+
+
+## Device (iotivity-lite) Keepalive to CoAP Gateway
+
+When a device is connected to coap-gateway and signed in, it will send a ping (a signal to check the connection) every 20 seconds. This delay is set by a defined macro [PING_DELAY](https://github.com/iotivity/iotivity-lite/blob/6ffd7aa0a259a59632de1e491131db242a62938a/api/cloud/oc_cloud_manager.c#L43).
+After sending a ping, the device will wait for a response (pong) for 1 second. If a response is not received within this time, the delay between pings will be increased to 4 seconds. If the device fails to receive a response for six consecutive pings, it will initiate a [reconnect process](/docs/architecture/component-overview/#reconnect-a-device).
```

Comment:
`the delay between pings will be increased to 4 seconds`

It's changed from 20secs to 4secs, that's a decrease and not an increase, no?